test cases/common/72 build always/version_gen.py

```diff
@@ -12,7 +12,7 @@ def generate(infile, outfile, fallback):
         (stdo, _) = p.communicate()
         if p.returncode == 0:
             version = stdo.decode().strip()
-    except:
+    except Exception:
```

Comment:
Should maybe change this to use `subprocess.check_output()` and handle `subprocess.CalledProcessError`?